nx.json

```diff
@@ -486,7 +486,7 @@
         },
         "core-time-picker": {
             "tags": ["fd"],
-            "implicitDependencies": ["core-utils", "core-time", "core-popover"]
+            "implicitDependencies": ["core-shared", "core-utils", "core-time", "core-popover"]
```

Comment:
Why is this needed? In PR for Angular 13 migration all implicit dependencies for core packages are being removed now

Reply:
the changes you are referencing are not merged yet

Reply:
reverted it